includes/class.llms.emails.php

```diff
@@ -327,18 +324,16 @@ public function get_table_html( $rows ) {
 
 		ob_start();
 		?>
-		<table style="<?php echo $table_style; ?>">
+		<table style="<?php echo esc_attr( $table_style ); ?>">
```

Comment:
How can we test this. I feel these styles and content may include things broken by the escaping functions.